src/components/Popover/PopoverContent.tsx

```diff
@@ -0,0 +1,155 @@
+import React, { MutableRefObject } from 'react';
+import styled, { css } from 'styled-components';
+import { usePopper } from 'react-popper';
+import type { Placement } from '@popperjs/core/lib/enums';
+
+import { theme } from '../../essentials/theme';
+import { get } from '../../utils/themeGet';
+import { Spaces } from '../../essentials';
+
+import { Card } from '../Card/Card';
+
+import { transition, useClickOutside } from './utils';
+
+interface PopoverContentProps {
+    /**
+     * Popover content (can be any valid React Element or component)
+     */
+    children: React.ReactNode;
+    /**
+     * Popover Trigger Reference
+     */
+    triggerRef: HTMLElement | HTMLDivElement | null;
+    /**
+     * Is Popover currently displayed?
+     */
+    isShown: boolean;
+    /**
+     * Optional: Specify the Popover content placement (placement changes automatically, if Popover content cannot fit inside viewport with selected placement)
+     */
+    placement?: Placement;
+    /**
+     * Optional: Specify the Popover content offset (margin between Popover trigger and content)
+     */
+    offset?: number;
+    /**
+     * Passes the onClose callback
+     */
+    onClose: (event?: React.KeyboardEvent<HTMLDivElement>) => void;
+}
+
+interface StyledPopoverProps extends React.CSSProperties {
+    isShown: boolean;
+    transform?: string;
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    translate?: any;
```

Comment:
What's the `translate` for?

Reply:
These relate to the properties that setup the positioning of Popover content dynamically in `<StyledPopoverParent>`...

This interface and especially the **translate: any** and **transform: string** lines had to be typed here in this fashion, otherwise the build was failing...

<img width="742" alt="Screenshot 2022-05-20 at 13 47 40" src="https://user-images.githubusercontent.com/14894844/169522006-670c80e0-57fb-42ed-b740-8598c7429331.png">

I couldn't for life set up types in a way that it wouldn't despite using the type the failing build normally describes for you, in a way such as `Types of property 'translate' are incompatible. Type 'string | number' is not assignable to type ''yes'' | ''no'`' ... 

Setting **translate** as **any** was the only way build was not failing, ( it was failing also with setting it as **unknown** or **never** ), therefore I felt like it was a case of _use **any** only and only if you cannot use anything else_ 🤷

Reply:
Do we understand from where we receive this `yes | no` thing? Maybe we should omit some property